hypervisor/arch/x86/trusty.c

```diff
@@ -106,9 +106,9 @@ static void create_secure_world_ept(struct vm *vm, uint64_t gpa_orig,
 	 */
 	sub_table_addr = alloc_paging_struct();
 	sworld_pml4e = HVA2HPA(sub_table_addr) | table_present;
-	mem_write64(pml4_base, sworld_pml4e);
+	*(uint64_t *)pml4_base = sworld_pml4e;
```

Comment:
why not use set_pgentry too?

Reply:
You're right. I'm not aware of this.
Thanks.